Makefile

```diff
@@ -2,13 +2,21 @@ start-deps:
 	docker-compose up -d
 	direnv reload
 
+start: start-deps
+	yarn build && node lib/entrypoint/graphql.js | yarn pino-pretty -c -l
```

Comment:
Don't you need to source `.envrc` here too?

Reply:
It works without it. Could you try it out on your machine too?